build/build.sh

```diff
@@ -9,18 +9,4 @@ export $(grep -v '^#' .env | xargs)
 (cd .. && docker build \
 --build-arg LIBRARY_RELEASE_NAME=$LIBRARY_RELEASE_NAME \
 --build-arg RUN_INTEGRATION_TESTS=$RUN_INTEGRATION_TESTS \
--t "$PROJECT_NAME" -f build/Dockerfile .)
-
-# Run a container to download artifacts
-docker run --name "${PROJECT_NAME}-build" "$PROJECT_NAME" /bin/true
-
-# Calculate test line coverage
-docker cp "${PROJECT_NAME}-build":/publish/reports/coverage.txt .
-LINE_COVERAGE=$(grep "Lines" -m 1 coverage.txt | cut -d " " -f 6 | sed "s/%//")
-rm -f coverage.txt
-
-# Update README.md file
-COVERAGE_BADGE="![Coverage $LINE_COVERAGE%](https://img.shields.io/badge/coverage-$LINE_COVERAGE%25-brightgreen)"
-sed -i "/Coverage /c $COVERAGE_BADGE" ../README.md
-
-docker rm "${PROJECT_NAME}-build"
\ No newline at end of file
+-t "$PROJECT_NAME" -f build/Dockerfile .)
```

Comment:
Does the container make sense now that coverage was remove?

It's still not documented what this is for 😅

Reply:
This doesn't run a container anymore, it only builds the image and all tests associated with it. That's documented [here](https://github.com/Languagewire/html-dumper/blob/main/README.md#development).